tests/drivers/rtc/rtc_api/prj.conf

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) 2022 Bjarki Arge Andreasen
 # SPDX-License-Identifier: Apache-2.0
-
-CONFIG_RTC=y
 CONFIG_ZTEST=y
 CONFIG_ZTEST_NEW_API=y
+
```

Comment:
The RTC API only tests the mandatory RTC features by default. Additional features like the alarm might not be implemented, which will lead to failed tests for those devices. This change shall be undone for that reason.

Reply:
The change was reverted/removed from the change set.